modules/era/lua_dynamis/globals/era_dynamis_spawning.lua

```diff
@@ -1295,7 +1295,6 @@ xi.dynamis.spawnDynamicPet =function(target, oMob, mobJob)
                 [true] =
                 {
                     ["Draklix Scalecrust"] = { "V. Wyvern", 27, 134, 0, 0, 714 }, -- Normal Vanguard's Wyvern (Vwyv)
-                    ["Wyrmwix Snakespecs"] = { "V. Wyvern", 27, 134, 0, 0, 714 }, -- Normal Vanguard's Wyvern (Vwyv)
```

Comment:
This entire section and up needs to be reverted. Names/changes in red need to stay